File: Uri/URI_CS/URI3299.cs
using System;

    class Program
    {
        static void Main(string[] args)
        {
            string num = Convert.ToString(Console.ReadLine());

            if (num.Contains("13")) {

                Console.WriteLine("{0} es de Mala Suerte",num);
            }
            else {
                Console.WriteLine("{0} NO es de Mala Suerte",num);
            }
        }    
} 

File: Uri/URI_JAVA/URI3299.java
package URI_JAVA;
import java.util.Scanner;

public class URI3299 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String num = scanner.nextLine();

        if (num.contains("13")) {
            System.out.println(num + " es de Mala Suerte");
        } else {
            System.out.println(num + " NO es de Mala Suerte");
        }
    }
}
